cmd/tsbs_generate_queries/uses/devops/groupby_orderby_limit.go

```diff
@@ -12,7 +12,9 @@ type GroupByOrderByLimit struct {
 
 // NewGroupByOrderByLimit returns a new GroupByOrderByLimit for given paremeters
 func NewGroupByOrderByLimit(core utils.DevopsGenerator) utils.QueryFiller {
-	return &GroupByOrderByLimit{core}
+	return &GroupByOrderByLimit{
```

Comment:
I'm not a huge fan of these changes, seems unnecessary?

Reply:
In general, it looks (at least for me) like filling `struct` data type in vertical form correlates with defining `struct` in the same vertical form. At least it seems to be easier to refer to `struct` fields when reviewing code. But, fill-one-field of the struct case can be treated separately - and may be it is easier to fill one field as a one line. However, as to multi-field structs - it really seems better to fill the vertically. So my proposition is to fill one-field structs as one-liners and multi-field - as multi-line.

Summary: this line updated, some other propose to keep multi-lined